scripts/cmake.sh

```diff
@@ -6,7 +6,7 @@ set -eux -o pipefail
 mkdir -p build
 
 # Generate build system specified in build directory with cmake
-cmake -Bbuild -GNinja
+cmake -Bbuild -GNinja -DCMAKE_EXPORT_COMPILE_COMMANDS=1
```

Comment:
We could use a CMakePresets.json to ensure this is always provided (we could also hide Ninja as the generator here, though I'd favour the "Ninja Muliti-Config" generator for only one build folder)

Reply:
Feel free to amend the PR. Sounds like a neat approach.

Reply:
We can improve this later. Merging now as I want this locally.